lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -2899,6 +2899,12 @@ def visit_FLOAT(self, type_, **kw):
     def visit_DOUBLE_PRECISION(self, type_, **kw):
         return "DOUBLE PRECISION"
 
+    def visit_DOUBLE(self, type_, **kw):
```

Comment:
remove

see basically people should use Double :)

Reply:
Removed